### PR TITLE
Add AwakenRegions for optimizing TiKV slow-score mechanism

### PR DESCRIPTION
**Pull request by LykxSassinator**
Signed-off-by: lucasliang nkcs_lykx@hotmail.com

## Description
As the title shown, this PR add an extra AwakenRegions into `pdpb.proto` as the basement for optimizing TiKV slow-score mechanism.

## Related ISSUE or PR
The related ISSUE can be found in TiKV ISSUE: [Optimize slow-score mechanism for reducing the time-cost on failure recovery.](https://github.com/tikv/tikv/issues/13648)

### CODE REVIEW

**Comment by LykxSassinator** [comment]
PTAL /cc @5kbpers

File: proto/pdpb.proto [hunk 1]
```diff
@@ -664,6 +664,13 @@ message RecoveryPlan {
     uint64 step = 6;
 }
 
+message AwakenRegions {
+    // Awake all regions if abnormal_stores is empty.
+    repeated uint64 abnormal_stores = 1;
+    // Target regions waited to be awaken.
+    repeated uint64 abnormal_regions = 2;
```
Comment (5kbpers) [inline proto/pdpb.proto:671]: Seems `abnormal_stores` is enough.

**Review by innerr — APPROVED** [review]
LGTM

**Pull request merged**